doc/man3/OSSL_sk_X509_add_cert.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+OSSL_sk_X509_add_cert,
+OSSL_sk_X509_add_certs -
+X509 certificate list handling functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/x509.h>
+
+ int OSSL_sk_X509_add_cert(STACK_OF(X509) *sk, X509 *cert,
+                           int up, int no_dup, int prepend);
+ int OSSL_sk_X509_add_certs(STACK_OF(X509) *sk, STACK_OF(X509) *certs, int up,
+                            int no_self_issued, int no_dups, int prepend);
+
+=head1 DESCRIPTION
+
+OSSL_sk_X509_add_cert() appends or prepends (depending on the I<prepend>
+argument) a certificate to the given list,
+optionally only if it is not already contained.
+On success the reference count of the certificate is increased
+if I<up> is nonzero.
```

Comment:
`no_self_issued` seems conspicuous by its absence in this function

Reply:
Yeah, this was due to a merge mistake while the parameter was renamed - fixed.